index.ts

```diff
@@ -238,8 +244,7 @@ app.prepare().then(() => {
   })
 
   server.use(router.routes())
-  server.listen(port, err => {
-    if (err) throw err
```

Comment:
Why are we ignoring errors now?

Reply:
According to koa types there is no error available in arguments